src/server/req_preemptjob.c

```diff
@@ -221,9 +236,8 @@ reply_preempt_jobs_request(int code, int aux, struct batch_request *local_preq)
 		index = 0;
 		count = 0;
 		preempt_jobs_list = NULL;
-	} else
-		return;
-
+	}
+	return;
```

Comment:
This is unnecessary, but doesn't harm anything.

Reply:
I wanted to remove it, don't know why I didn't :)